doc/environment.yml

```diff
@@ -0,0 +1,73 @@
+name: doc
```

Comment:
Maybe rename this file to `conda_environment.yml` to make clearer what it is about?